andela/rainier-rc
# #151994135 Real time search with autocomplete

#### What does this PR do?
Have users get search results real time while they search

#### How should this be manually tested?
- Run the application using `reaction` command
- Go to the url `http://localhost:3000`
- On the navigation bar, click on the search icon
- An overlay modal appears with a search box
- Search for products and get search results while typing
#### Any background context you want to provide?
Used an atmosphere autocomplete package `mizzao: autocomplete`
#### What are the relevant pivotal tracker stories?
Story Id: 151994135
Story type: Feature
Story Title: Display real time search results as the shopper is typing
#### Screenshots (if appropriate)
<img width="1387" alt="screen shot 2017-10-27 at 9 16 25 am" src="https://user-images.githubusercontent.com/24405172/32094423-988a7d14-baf7-11e7-9747-03564c4cfd5e.png">

#### Questions:
None
